functions.yml

```diff
@@ -271,8 +282,9 @@ quality_control:
       args:
         - name: genotypes
           doc: The array of genotype structs
```

Comment:
An array of genotype structs with `calls` field

Reply:
Done